LibraBFT/Base/KVMap.agda

```diff
@@ -59,6 +59,8 @@ module LibraBFT.Base.KVMap  where
    elems          : KVMap Key Val → List Val
    delete         : Key → KVMap Key Val → KVMap Key Val
    singleton      : Key → Val → KVMap Key Val
+   fromList       : List (Key × Val) → KVMap Key Val
```

Comment:
Actually, I did not duplicate because of not liking names (which is true).
I did not see the existing function.
I am not going to make a change in this PR.
There is an existing "cleanup KVMap" item on our TODO list.